Vic2ToHoI4/Data_Files/configurables/names.txt

```diff
@@ -6216,4 +6356,716 @@ vandal = {
 	weapon_companies = { Rheinmetall Mauser Opel }
 	aircraft_companies = { Messerschmitt Junkers "Focke Wulf" Heinkel Dornier }
 	intelligence_agencies = { "Vandalismus" }
-}
\ No newline at end of file
+}
+
+zzz_religion_cybelene = {
+ female_names = { " " }
```

Comment:
Make this and all other ones where you don't have any names like this:
```
female_names = {}
```
The converter can use that to try and find generic localisations instead.

Reply:
Those that start with zzz_religion_ are meant to be empty and never appear. They're the way we have Vic2 display a country's religion, but they should never be primary cultures.

Reply:
Ah, gotcha!